src/test_l1_tvSettings.c

```diff
@@ -12980,11 +12954,15 @@ void test_l1_tvSettings_positive_GetCurrentComponentSaturation (void)
 	result = TvInit();
 	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
 
-	/* Step 02: Calling tvsettings GetCurrentComponentSaturation and expecting the API to return success */
-	result = GetCurrentComponentSaturation((tvDataComponentColor_t)Configfile.componentColor.modeId[0],&saturation);
+	/* Step 02: Calling tvsettings to Set the SetCMSState for value true and the API to return success */
+	result = SetCMSState(true);
```

Comment:
I think this SaveCMSState(true) has to be for SetCurrentComponentSaturation and not for GetCurrentComponentSaturation

Reply:
moved to setCurrentComponentHue